resources/lang/en/beatmaps.php

```diff
@@ -130,18 +130,19 @@
     ],
 
     'nominations' => [
-        'disqualifed-at' => 'Disqualified :time_ago (:reason).',
-        'disqualifed_no_reason' => 'no reason specified',
-        'disqualification-prompt' => 'Reason for disqualification?',
+        'disqualification_prompt' => 'Reason for disqualification?',
+        'disqualified_at' => 'Disqualified :time_ago (:reason).',
+        'disqualified_no_reason' => 'no reason specified',
         'disqualify' => 'Disqualify',
         'incorrect_state' => 'Error performing that action, try refreshing the page.',
         'nominate' => 'Nominate',
-        'nominated-by' => 'nominated by :users',
-        'nominate-confirm' => 'Nominate this beatmap?',
+        'nominate_confirm' => 'Nominate this beatmap?',
+        'nominated_by' => 'nominated by :users',
         'qualified' => 'Estimated to be ranked :date, if no issues are found.',
-        'qualified-soon' => 'Estimated to be ranked soon, if no issues are found.',
-        'reset-confirm' => 'Are you sure? Posting a new problem will reset nominations.',
+        'qualified_soon' => 'Estimated to be ranked soon, if no issues are found.',
         'required_text' => 'Nominations: :current/:required',
+        'reset_at' => 'Reset :time_ago by discussion :discussion.',
```

Comment:
probably worth being more explicit here and saying `Nominations reset :time_ago by...`